tests/ui/UnreadIndicatorsTest.js

```diff
@@ -25,6 +25,8 @@ beforeAll(() => {
     // behavior. But here we just want to treat all API requests as a generic "success" and in the cases where we need to
     // simulate data arriving we will just set it into Onyx directly with Onyx.merge() or Onyx.set() etc.
     global.fetch = TestHelper.getGlobalFetchMock();
+
+    jest.setTimeout(15000);
```

Comment:
Why is there here and why is it so big?

Reply:
It was discussed here - https://github.com/Expensify/App/pull/10986#discussion_r981413650

Reply:
OK, thank you. Can you please do another PR to add a code comment for why it's there, and why it has the value 15000?

Reply:
Separately, I have started to see these tests time out in the CI so maybe we need to make it bigger or explore optionally not lazy loading the screens the tests (though not sure how easy it will be to do the latter)

Reply:
cc @Szymon20000 for your awareness